helper/date_format_test.go

```diff
@@ -18,6 +18,25 @@ func TestDateFormat(t *testing.T) {
 	})
 }
 
+func TestTimeFormatHH(t *testing.T) {
+	Convey("Time format returns time in 24h when passed date-time", t, func() {
+		So(helper.TimeFormatHH("2019-05-21T23:00:00.000Z"), ShouldEqual, "00:00") // BST
+		So(helper.TimeFormatHH("2019-05-21T13:40:00.000Z"), ShouldEqual, "14:40") // BST
+		So(helper.TimeFormatHH("2022-11-17T22:00:00.000Z"), ShouldEqual, "22:00") // GMT
```

Comment:
We have 2 assertions for BST and 2 for GMT in the other test but only 2 and 1 here. I'm not sure what having multiple dates adds but maybe make them consistent?